src/cosmos/Swap.fixture.tsx

```diff
@@ -8,18 +12,14 @@ import {
   SUPPORTED_LOCALES,
   SupportedChainId,
   SwapWidget,
-} from '@uniswap/widgets'
-import { useEffect } from 'react'
-import { useValue } from 'react-cosmos/fixture'
-
-import { DAI, USDC_MAINNET } from '../constants/tokens'
+} from '../index'
```

Comment:
this seems fine, but curious what the change gets us?

Reply:
`@uniswap/widgets` is self-referential, so it's outside of the type system. I can probably add it as a path, though, which might make this look a little nicer.